tests/test_orders.py

```diff
@@ -205,3 +206,13 @@ def test_list_order_refund(client, response):
     order = client.orders.get(ORDER_ID)
     refunds = order.refunds
     assert_list_object(refunds, Refund)
+
+
+def test_list_shipments(client, response):
```

Comment:
This can now also be a property test  inside `test_get_order`. Same for the order.refunds test a few line up.